locations/spiders/screwfix.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+
+class ScrewfixSpider(scrapy.Spider):
+	name = "screwfix"
```

Comment:
Please use four spaces instead of a tab.

Reply:
Sorry, I forgot. I'll change to spaces.